src/wikibase/mod.rs

```diff
@@ -46,19 +46,18 @@ impl Config {
     }
 }
 
-pub fn new(config_file: &str) -> Result<Client, Error> {
-    let mut f = std::fs::File::open(config_file)?;
-    let mut content = String::new();
-    f.read_to_string(&mut content)?;
-    let config = toml::from_str::<Config>(&content)?;
-
-    Ok(Client {
-        api: client::new(config.clone()),
-        sparql: sparql_client::new(config),
-    })
-}
-
 impl Client {
+    pub fn new(config_file: &str) -> Result<Self, Error> {
```

Comment:
completely minor too: you could have made it a bit more generic:
```rust
pub fn new<P: AsRef<Path>>(config_file: P) -> Result<Self, Error> {
}
```
(it's a bit more idiomatic)